pkg/winch/auth.go

```diff
@@ -42,9 +47,13 @@ func (f *AuthFactory) Get(configSource *pb.AuthSource) (tokenauth.Source, error)
 	var source tokenauth.Source
 	var err error
 
+	// Get is always use on startup only, so we can set reasonable timeout here.
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
This should probably be a flag, might be erroring out on slow networks (?)

Reply:
yea